templates/main.php

```diff
@@ -51,6 +51,7 @@
                 ng-class="{ active: note.id == route.noteId }">
                 <a href="#/notes/{{ note.id }}">
                     {{ note.title | noteTitle }}
+                    <span ng-if="note.unsaved">*</span>
```

Comment:
Could be solved with `ng-class` on the link instead. This way the * or any other styling could be injected by css (::after).